src/platform_impl/ios/view.rs

```diff
@@ -197,6 +296,43 @@ impl WinitView {
         this
     }
 
+    pub(crate) fn recognize_pinch_gesture(&self, should_recognize: bool) {
+        if should_recognize && self.ivars().pinch_gesture_recognizer.borrow().is_none() {
+            let pinch = UIPinchGestureRecognizer::init_with_target(self, sel!(pinchGesture:));
+            self.addGestureRecognizer(pinch.as_super());
+            self.ivars().pinch_gesture_recognizer.replace(Some(pinch));
+        } else if let Some(recognizer) = self.ivars().pinch_gesture_recognizer.take() {
+            self.removeGestureRecognizer(&recognizer);
+        }
+    }
+
+    pub(crate) fn recognize_doubletap_gesture(&self, should_recognize: bool) {
+        if should_recognize && self.ivars().doubletap_gesture_recognizer.borrow().is_none() {
+            let tap = UITapGestureRecognizer::init_with_target(self, sel!(doubleTapGesture:));
+            unsafe {
+                let _: () = msg_send![&tap, setNumberOfTapsRequired: 2_u64];
+                let _: () = msg_send![&tap, setNumberOfTouchesRequired: 1_u64];
+            }
```

Comment:
Use the methods you've defined in the `uikit` module:
```suggestion
            tap.setNumberOfTapsRequired(2);
            tap.setNumberOfTouchesRequired(1);
```